src/plugins/eos-external-apps-build-install

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash -e
+
+# eos-external-apps-build-install - Build an external app and install it in a system repo
+# Copyright (C) 2016 Endless Mobile, Inc.
+#
+# Authors:
+#	Joaquim Rocha <jrocha@endlessm.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+function help {
+    echo "$0"
+    echo "Usage:"
+    echo "  $0 SYSTEM_REPO APP_NAME EXTERNAL_ASSET_URL EXTERNAL_ASSET_TYPE"
+    echo ""
+    echo "APP_NAME              the app name (e.g. org.myorg.MyApp)"
+    echo "EXTERNAL_ASSET_URL    the asset URL (e.g. https://www.myorg.org/MyOrg_0.1.2-1.deb)"
+    echo "EXTERNAL_ASSET_TYPE   the type of the asset ('deb' or 'tar')"
+}
+
+SYSTEMREPO=$1
+APPNAME=$2
+ASSETURL=$3
+ASSETTYPE=$4
+
+if [ -z "$SYSTEMREPO" ]; then
+   echo "$0: Need the system repo name"
+   help
+   exit 1
+fi
+
+if [ -z "$APPNAME" ]; then
+   echo "$0: Need the app name"
+   help
+   exit 1
+fi
+
+if [ -z "$ASSETURL" ]; then
+    echo "$0: Need an external asset URL"
+    help
+    exit 1
+fi
+
+if [ -z "$ASSETTYPE" ]; then
+    echo "$0: Need the external asset type"
+    help
+    exit 1
+fi
+
+EXTAPPSDIR="/opt/flatpak-external-apps"
+APPDIR="$EXTAPPSDIR/$APPNAME"
+BUILDDIR="$APPDIR/app"
```

Comment:
Calling this `app` is a bit confusing, since that's usually the prefix that would be inside `BUNDLEDIR`... How about just calling it `build` instead?